examples/python/queries/classification.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import _dummy_objects
+
+# Declare a floating point value representing the occupancy posterior in log
+# odds as queried from the map in one of the previous examples
+occupancy_log_odds = _dummy_objects.example_occupancy_log_odds()
+
+# A point is considered unobserved if its occupancy posterior is equal to the
+# prior. Wavemap assumes that an unobserved point is equally likely to be
+# free or occupied. In other words, the prior occupancy probability is 0.5,
+# which corresponds to a log odds value of 0.0. Accounting for numerical
+# noise, checking whether a point is unobserved can be done as follows:
+kUnobservedThreshold = 1e-3
+is_unobserved = np.abs(occupancy_log_odds) < kUnobservedThreshold
+print(is_unobserved)
+
+
+# In case you would like to convert log odds into probabilities, we provide
+# the following convenience function:
+def logOddsToProbability(log_odds):
```

Comment:
Naming not "correct" Python :-)

Reply:
Thanks, I'll switch all the Python method names to snake case.